spyder/config/main.py

```diff
@@ -78,12 +78,16 @@
               'custom_margin': 0,
               'use_custom_cursor_blinking': False,
               'show_internal_errors': True,
-              'check_updates_on_startup': True,
               'cursor/width': 2,
               'completion/size': (300, 180),
               'report_error/remember_token': False,
               'show_dpi_message': True,
               }),
+            ('update_manager',
+             {
+              'check_updates_on_startup': True,
+              'check_stable_only': True,
+              }),
```

Comment:
I think this requires a `CONF_VERSION` bump?: https://github.com/spyder-ide/spyder/blob/9f1baeb72fae2214f30bccf44aa4aa0c32b7f122/spyder/config/main.py#L660-L670

Reply:
Good catch @dalthviz. `check_updates_on_startup` is being _removed_ from the `main` section, so I think you are correct and, according to note 2, a major version bump is required. I'll make that change.